doc/src/actioncode.md

```diff
@@ -10,11 +10,12 @@ Action code is normal Rust code with the addition of the following special varia
 
  * `$lexer` allows access to the lexer and its [various
    functions](https://softdevteam.github.io/grmtools/master/api/lrpar/trait.Lexer.html).
-   The most commonly used of these is the `lexeme_str` function, which allows
-   us to turn `Lexeme`s into `&'input str`s representing the corresponding
-   portion of the user's input. As this may suggest, actions may also reference the
-   special lifetime `'input` (without any `$` prefix), which allows strings to
-   be returned / stored by the grammar without copying memory.
+   The most commonly used of these is the `span_str` function, which allows us
```

Comment:
Better: The most common use for this is...

Reply:
It's referring to the lexer's functions, so I think the current phrasing is OK?

Reply:
Ah, I see it now. Ok that's fine then.